src/routes/article-list.ts

```diff
@@ -2,9 +2,14 @@ import Router from '@koa/router';
 import { Context, Middleware, Next } from 'koa';
 import { hydra, rdf, schema } from 'rdf-namespaces';
 import Routes from './index';
+import Articles from '../adaptors/articles';
 
-export default (router: Router): Middleware => (
+export default (articles: Articles, router: Router): Middleware => (
   async ({ request, response }: Context, next: Next): Promise<void> => {
+    const list = [];
+    for await (const article of articles) {
```

Comment:
With the current lint rules this triggers an error:

```
/app/src/routes/article-list.ts
  10:5  error  iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations  no-restricted-syntax
```

Should we consider adjusting our rules to allow for loops?